packages/web-components/fast-components/src/avatar/avatar.definition.ts

```diff
@@ -0,0 +1,62 @@
+import { WebComponentDefinition } from "@microsoft/fast-tooling/dist/data-utilities/web-component";
+import { DataType } from "@microsoft/fast-tooling";
+
+export const fastAvatarDefinition: WebComponentDefinition = {
+    version: 1,
+    tags: [
+        {
+            name: "fast-avatar",
+            description: "The FAST avatar element",
+            attributes: [
+                {
+                    name: "initials",
+                    description: "The initials attribute",
```

Comment:
For all the descriptions, can we use something more informative? Something like "Person's initials to show in place of a profile image".